packages/storybook/src/nimble/button/button.stories.ts

```diff
@@ -22,6 +23,7 @@ interface ButtonArgs {
     icon: boolean;
     contentHidden: boolean;
     endIcon: boolean;
+    click: () => void;
```

Comment:
I'm feeling deja vu but why does this need to be a function? Can it just be undefined?

Reply:
It was just a mindless copy-paste pattern. Changed to undefined.